File: Basic_Math/divide_by_two/DivideByTwo.java
package Basic_Math.divide_by_two;

public class DivideByTwo {
    public static void main(String[] args) {
        int number = 10;
        System.out.println("Number of times " + number + " can be divided by two: " + countDivisionsByTwo(number));
    }

    public static int countDivisionsByTwo(int n) {
        int count = 0;
        while (n > 1) {
            n = n / 2;
            count++;
        }
        return count;
    }
}

File: Basic_Math/divide_by_two/divideByTwo.go
package main

import "fmt"

func dividByTwo(n int) int {
	count := 0
	for n > 1 {
		n = n / 2
		count++
	}
	return count
}

func main() {
	fmt.Println("divide by two", dividByTwo(20))
}
